src/lib/core/utils/axis-label-unit.ts

```diff
@@ -0,0 +1,15 @@
+import { Variable } from '../types/study';
+
+export function axisLabelUnit(
```

Comment:
Would you mind renaming this `axisLabelWithUnit` ?

Reply:
@bobular No problem! I can change the function name :)

Reply:
@bobular I have changed the name as suggested